src/Avalonia.Visuals/Media/HsvColor.cs

```diff
@@ -0,0 +1,493 @@
+        /// </summary>
+        /// <param name="hsvColor">The color in the HSV color model.</param>
+        /// <returns>A new RGB <see cref="Color"/> equivalent to the given HSVA values.</returns>
+        public static Color ToRgb(HsvColor hsvColor)
+        {
+            return HsvColor.ToRgb(hsvColor.H, hsvColor.S, hsvColor.V, hsvColor.A);
+        }
+
+        /// <summary>
+        /// Converts the given HSVA color channel values to it's RGB color equivalent.
+        /// </summary>
+        /// <param name="hue">The hue channel value in the HSV color model in the range from 0..360.</param>
+        /// <param name="saturation">The saturation channel value in the HSV color model in the range from 0..1.</param>
+        /// <param name="value">The value channel value in the HSV color model in the range from 0..1.</param>
+        /// <param name="alpha">The alpha channel value in the range from 0..1.</param>
+        /// <returns>A new RGB <see cref="Color"/> equivalent to the given HSVA values.</returns>
+        public static Color ToRgb(
+            double hue,
+            double saturation,
+            double value,
+            double alpha = 1.0)
+        {
+            // Note: Conversion code is originally based on the C++ in WinUI (licensed MIT)
+            // https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/Common/ColorConversion.cpp
+            // This was used because it is the best documented and likely most optimized for performance
+            // Alpha channel support was added
+
+            // We want the hue to be between 0 and 359,
+            // so we first ensure that that's the case.
+            while (hue >= 360.0)
+            {
+                hue -= 360.0;
+            }
+
+            while (hue < 0.0)
+            {
+                hue += 360.0;
+            }
+
+            // We similarly clamp saturation and value between 0 and 1.
+            saturation = saturation < 0.0 ? 0.0 : saturation;
+            saturation = saturation > 1.0 ? 1.0 : saturation;
+
+            value = value < 0.0 ? 0.0 : value;
+            value = value > 1.0 ? 1.0 : value;
+
+            // The first thing that we need to do is to determine the chroma (see above for its definition).
+            // Remember from above that:
+            //
+            // 1. The chroma is the difference between the maximum and the minimum of the RGB channels,
+            // 2. The value is the maximum of the RGB channels, and
+            // 3. The saturation comes from dividing the chroma by the maximum of the RGB channels (i.e., the value).
+            //
+            // From these facts, you can see that we can retrieve the chroma by simply multiplying the saturation and the value,
+            // and we can retrieve the minimum of the RGB channels by subtracting the chroma from the value.
+            double chroma = saturation * value;
+            double min = value - chroma;
```

Comment:
```suggestion
            var chroma = saturation * value;
            var min = value - chroma;
```

Reply:
I strongly prefer not to overuse `var` when it isn't explicitly clear. I know you prefer to use `var` for everything. `double` makes the type clear here. Types are actually quite important in this sort of code. Other places use `byte`.

Reply:
Well, this isn't worth debating here as long as I can keep the type clear elsewhere. Since this uses defined variables directly it can be inferred easily when reading through the full algorithm.